src/hotspot/share/opto/cfgnode.hpp

```diff
@@ -318,11 +319,23 @@ class MultiBranchNode : public MultiNode {
 //------------------------------IfNode-----------------------------------------
 // Output selected Control, based on a boolean test
 class IfNode : public MultiBranchNode {
+ public:
+  float _prob;                           // Probability of true path being taken.
+  float _fcnt;                           // Frequency counter
```

Comment:
Moved these fields up to the start of the class to have them together with the new field.